File: Chapter-2/Java/Problem4.java

//Import required libraries
import java.util.Scanner;

public class Problem4 {
    public static void main(String[] args) {
        // Declare variables and take input
        double t, v, wcf;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter temperature: ");
        t = sc.nextDouble();
        System.out.println("Enter velocity: ");
        v = sc.nextDouble();
        // Calculate the result
        wcf = 35.74 + (0.6215 * t) + ((0.4275 * t) - 35.75) * Math.pow(v, 0.16);
        // display the inputs and results
        System.out.println("Temperature: " + t);
        System.out.println("Velocity: " + v);
        System.out.println("Wind Chill Factor: " + wcf);
        sc.close();
    }
}

File: Chapter-2/Python/Problem4.py
t = float(input("Enter temperature: "))
v = float(input("Enter velocity: "))
wcf = 35.74+(0.6215*t) +((0.4275*t) - 35.75)*(v**0.16)

print("Temperature: %.3f" %t)
print("Velocity: %.3f" %v)
print("Wind Chill Factor: %.3f" %wcf)